### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by SashaViun**
[DEMO LINK](https://SashaViun.github.io/react_movies-list/)

### CODE REVIEW

**Comment by NVyrsta** [comment]
The demo link doesn't work

File: src/App.tsx [hunk 1]
```diff
@@ -3,97 +3,13 @@ import React from 'react';
 
 import './App.scss';
 // import moviesFromServer from './api/movies.json';
```
Comment (NVyrsta) [inline src/App.tsx:5]: redundant comment

**Review by NVyrsta — CHANGES_REQUESTED** [review]
Good job 👍
Just fix the demo link

**Review by ArturErmolenko — APPROVED** [review]
GJ 👍

**Pull request closed without merging**